### ISSUE TITLE
Grunt deploy failed continued issue from #731

### ISSUE THREAD

**Issue by deeforte** [ga-wdi-boston/full-stack-project#735]
Apologize for opening another issue but I am not able to grunt deploy successfully.  I did as Jordan suggested on issue 731 and add & committed the files.  I was able to switch to master but now am trying to grunt deploy from master.
Do I need to push to origin gh-pages as well?   That does not seem right to me and want to check with you before making that step.

I did as Ben suggested and researched the game project issues.  Looks like Joni had one close to mine but the resolution is not in the issue tracker.

Ideally I'd like to get back to where I was before I entered grunt deploy from the wrong branch.  I did a commit/push to apiconnect prior to do that.

Can I get a couple minutes of someone's time?    Thank you!
----------------------------------
Description of issue 731 is below.
When working on my client I made a mistake and used 'grunt deploy' on the wrong branch. The grunt deploy errorred, but now I am not able get out of the gh-pages branch because of untracked work.
Below is the error I am seeing. I want to totally remove the gh-pages branch.

~/wdi/projects/stash-client (gh-pages)
$ git checkout apiconnect
error: The following untracked working tree files would be overwritten by checkout:
node_modules/.bin/acorn
node_modules/.bin/babylon
node_modules/.bin/browserslist..........
node_modules/JSV/jsdoc-toolkit/app/lib/JSDOC/Token.js
node_modules/JSV/jsdoc-toolkit/app/lib/JSDOC/TokenReader.js
node_modules/JSV/jsdoc-toolkit/ap
Aborting
~/wdi/projects/stash-client (gh-pages)

**Comment by danman01** [comment]
Hey @deeforte let's grab some time to go over this.

**Comment by deeforte** [comment]
Thanks.  Let me know when is good for you.

**Comment by deeforte** [comment]
Thanks for your time Danny!

**Comment by danman01** [comment]
Just to report what happened, we needed to base the current project on `browser-template`, and to do this we rebased. We also got some good practice fixing merge conflicts, and then aborting a rebase! `grunt deploy` was stopping 4/5ths of the way through, which led us to look at the browser-template Grunt `aliases.json` file and dig into what grunt deploy was actually doing. Running the deploy script line by line led us to the error.
